stl/inc/yvals_core.h

```diff
@@ -51,13 +52,16 @@
 // P0898R3 Standard Library Concepts
 // P0919R3 Heterogeneous Lookup For Unordered Containers
 // P0966R1 string::reserve() Should Not Shrink
+// P1024R3 Enhancing span Usability
+// P1085R2 Removing span Comparisons
 // P1209R0 erase_if(), erase()
 // P1227R2 Signed std::ssize(), Unsigned span::size()
-//     (partially implemented)
 // P1357R1 is_bounded_array, is_unbounded_array
+// P1394R4 Range constructor for std::span

```

Comment:
Nitpick, this should be `P1394R4 Range Constructor For span` (as we capitalize everything but identifers, and we avoid `std::` qualification in titles except in special circumstances).

Reply:
Done